dev/io.openliberty.microprofile.lra.1.0.internal_fat_tck/bnd.bnd

```diff
@@ -0,0 +1,34 @@
+#*******************************************************************************
+# Copyright (c) 2020 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+#*******************************************************************************
+-include= ~../cnf/resources/bnd/bundle.props
+bVersion=1.0
+
+javac.source: 1.8
+javac.target: 1.8
+
+src: \
+	fat/src
+
+fat.project: true
+
+tested.features: mpLRA-1.0
```

Comment:
FYI,  you don't need to list features here which are in your server xml.

`tested.features` is to tell the build infrastructure that your test uses additional features not listed in server.xml (usually through `@RepeatTests`)